ocaml/xapi-idl/cluster/cluster_interface.ml

```diff
@@ -271,4 +272,129 @@ module LocalAPI (R : RPC) = struct
       @-> enabled_p
       @-> returning unit_p err
       )
+
+  type dict = (string * string) list [@@deriving rpcty]
+
+  module TypeCombinators = struct
```

Comment:
Looks like we have an identical module in storage_interface.ml, xenops_interface.ml and now here. It's worth putting it somewhere more central to avoid duplication. E.g. a file `ocaml/xapi-idl/lib/type_combinators.ml`.

Reply:
indeed, and in xenops_interface.ml as well, I will remove the duplicate code

Reply:
done